test_unstructured_ingest/src/mongodb.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRC_PATH=$(dirname "$(realpath "$0")")
+SCRIPT_DIR=$(dirname "$SRC_PATH")
+cd "$SCRIPT_DIR"/.. || exit 1
+echo "SCRIPT_DIR: $SCRIPT_DIR"
+OUTPUT_FOLDER_NAME=mongodb
+OUTPUT_ROOT=${OUTPUT_ROOT:-$SCRIPT_DIR}
+OUTPUT_DIR=$OUTPUT_ROOT/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$OUTPUT_ROOT/workdir/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+max_processes=${MAX_PROCESSES:=$(python3 -c "import os; print(os.cpu_count())")}
+SOURCE_MONGO_COLLECTION="sample-mongodb-data"
+CI=${CI:-"false"}
+
+if [ -z "$MONGODB_URI" ] && [ -z "$MONGODB_DATABASE_NAME" ]; then
```

Comment:
Can this be tested via docker?

Reply:
@rbiseck3 Definitely. But the dest connector was with the hosted MongoDB. How do we decide which to use a Docker or the hosted?

Reply:
cc @rbiseck3 @potter-potter